### ISSUE TITLE
NVDA does not announce the number of child items 

### ISSUE THREAD

**Issue by anOolamr** [nvaccess/nvda#13891]
NVDA does not announce the number of child items
- a collapsed node is expanded
- the keyboard focus navigates to an expanded node

See also bug report #13386. 

### Steps to reproduce:

1. If not available, download the SwingSet2 Demonstration Program from: https://docs.oracle.com/javase/tutorial/uiswing/lookandfeel/plaf.html#swingset2

2. Start NVDA.

3. Launch SwingSet2.jar with an up-to-date Java JRE.
-> The SwingSet2 application window appears.

4. Navigate with arrow right key to the "JTree demo" radio button and press spacebar to select it.

5. Press Tab key twice to navigate to the JTree control.
-> NVDA announces the "Tree" role of the JTree control.

6. Press arrow down key once to select the root node.
-> NVDA announces the name "Music", some state information ("selected, expanded, editable") and the position ("1 of 1, level 1") of the root node.
-> Missing announcement: number of its child items ("3 items").

7. Press arrow down key once to select the first child item of the root node.
-> NVDA announces the level ("level 2"), the name "Classical", some state information ("selected, collapsed, editable") and the position ("1 of 3") of the node. 

8. Press arrow right to expand the "Classical" tree node.
-> NVDA announces nothing. NVDA should announce the name and the state change ("expanded") of the node and the number of its child items ("3 items").

Note: The missing expand/collapse announcement is already described in bug report #13623. 

### Actual behavior:
NVDA does not announce the number of child items.

>level 5 No. 1 - C selected, editable 1 of 5

### Expected behavior:
NVDA announces also the number of child items when
- a collapsed node is expanded
- the keyboard focus navigates to an expanded node

>level 5 No. 1 - C selected, editable 1 of 5  3 child items

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
Version: 2022.1

#### Windows version:
Version	10.0.19043 Build 19043

#### Name and version of other software in use when reproducing the issue:
java version "18.0.1" 2022-04-19
Java(TM) SE Runtime Environment (build 18.0.1+10-24)
Java HotSpot(TM) 64-Bit Server VM (build 18.0.1+10-24, mixed mode, sharing)

#### Other information about your system:
Java Access Bridge is enabled.

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes. Same behavior.

#### If NVDA add-ons are disabled, is your problem still occurring?
Not applicable.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not applicable.

**Comment by seanbudd** [comment]
Is the reasoning for this the same as #13892

**Comment by Adriani90** [comment]
This is indeed not needed because it adds extra unneeded verbosity. When you expand an item and press down arrow key, you already will hear the number of the child elements. Closing as won't fix.
